cmd/version/version.go

```diff
@@ -18,10 +18,11 @@ func GetVersionCmd() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			ctx := context.TODO()
 			v := cautils.NewIVersionCheckHandler(ctx)
-			v.CheckLatestVersion(ctx, cautils.NewVersionCheckRequest(cautils.BuildNumber, "", "", "version"))
+			versionCheckRequest := cautils.NewVersionCheckRequest(cautils.BuildNumber, "", "", "version")
+			v.CheckLatestVersion(ctx, versionCheckRequest)
 			fmt.Fprintf(os.Stdout,
 				"Your current version is: %s\n",
-				cautils.BuildNumber,
+				versionCheckRequest.ClientVersion,
```

Comment:
can you add a unit test somehow to qualify for the bounty?

Reply:
Sure will work on it

Reply:
@matthyx I have added a test to evaluate the stdout, but I also had to edit the function `GetVersionCmd` to use cobra's stdout (simpler when testing) instead of os. I hope that's good for you